styles/styles.css

```diff
@@ -55,5 +63,51 @@ hr {
 }
 
 .text-adv {
+  font-weight: 400;
   font-size: 24px;
+  line-height: 28px;
+}
+
+.footer {
+  width: 1440px;
+  height: 625px;
+  margin: 0 auto;
+  padding: 120px 270px 0 104px;
+}
+
+.footer-background {
+  width: 100%;
+  background-color: #10367c;
+}
+
+.xs-footer {
```

Comment:
Im not actually sure about "xl"/"xs" naming. Its look like responsive classes, but you dont now it yet... 
try, maybe for now call classes - parent/children
like:
<div class="footer" or "footer-container">
  <div class="footer-content" >
    <>...some content...<>
  </div>
</div>